src/Domain/Infra/DependencyInjection/BundleHelper.php

```diff
@@ -211,6 +211,7 @@ private static function initDoctrineOrm(ContainerBuilder $container): void
         if (FeatureDetection::hasFrameworkBundle($container)) {
             $container->register(DoctrineInfra\MappingCacheWarmer::class)
                 ->setPublic(false)
+                ->setArgument('$rootDir', '%kernel.project_dir%')
```

Comment:
only BundleHelper knows about the project folder semantics, so here we should specify `%kernel.project_dir%/config/packages/msgphp/doctrine`

im still a bit skeptical about this magic path hardcoded, but it's convenient for sure. And overriding during cache warmup is fine also :)